test/unit/object/test_file.py

```diff
@@ -41,6 +42,58 @@ def test_delete_file(test_file, mock_box_session, etag, if_match_header):
     )
 
 
+def test_create_task(test_file, test_task, mock_box_session):
+    # pylint:disable=redefined-outer-name
+    expected_url = "{0}/tasks".format(API.BASE_API_URL)
+    expected_body = {
+        'item': {
+            'type': 'file',
+            'id': '42',
+        },
+        'action': 'review',
+        'message': 'Test Message',
+        'due_at': '2014-04-03T11:09:43-07:00',
```

Comment:
You might want to extract these values into variables to make it easier to put them in all three pleaces (this `expected_body`, the `return_value` below, and the actual invocation of `create_task()` below that).